test/native/symbolsLinuxTest.cpp

```diff
@@ -60,4 +60,15 @@ TEST_CASE(MappedTwiceAtZeroOffset) {
     hello();
 }
 
+TEST_CASE(MultipleMatchingSymbols) {
+    const void* sym = resolveSymbol("multiplematching.so", "Class::function");
+    ASSERT(sym);
+
+    const void* symOk = resolveSymbol("multiplematching.so", "_ZN5Class8functionEv");
```

Comment:
`sym_ok` and `sym_cold`

Reply:
4fbb4e1dd15c3a4ccfb26e60469aea1abdda6534